Packs/CommonScripts/Scripts/Jmespath/README.md

```diff
@@ -0,0 +1,2 @@
+## Jmespath
+This is a transformer script that allows the use of Jmespath searching on JSON objects.
```

Comment:
This should be in the description file.
Please add here the documentation for the script.
Use demisto-sdk generate-docs
https://github.com/demisto/demisto-sdk/tree/master/demisto_sdk/commands/generate_docs

Reply:
This is a transformer script and not designed to be used manually (i.e. !jmespath etc), therefore the generate-docs will not work for this script.

Reply:
It doesn't matter, we now require a readme for all content.
You can run the script without an example so it will generate the general info for it